nixos/modules/programs/ssh.nix

```
@@ -154,12 +154,21 @@ in
             };
             hostNames = mkOption {
               type = types.listOf types.str;
-              default = [];
+              default = [ name ] ++ config.extraHostNames;
```

surprising that the manual built with this, this file should be docs-sandboxed :thinking: looks like submodule `config` arguments aren't restricted properly.
```suggestion
              default = [ name ] ++ config.extraHostNames;
              defaultText = literalExpression "[ ${name} ] ++ config.${options.extraHostNames}";
```
in the manpage this renders as:
```
Default: [ ‹name› ] ++ config.programs.ssh.knownHosts.<name>.extraHostNames
```